web-client/src/ducks/timeline/functions.ts

```diff
@@ -1,15 +1,114 @@
 import { firestore } from 'src/firebase';
-import { TimelineItemFirestoreConverter } from 'src/models/requests/timeline';
+// eslint-disable-next-line no-unused-vars
+import { IOffer, Offer } from 'src/models/offers';
+import { IRequest, Request, RequestFirestoreConverter, RequestRefType } from 'src/models/requests';
+import { IRequestWithOffersAndTimelineItems, RequestWithOffersAndTimeline } from 'src/models/requests/RequestWithOffersAndTimeline';
+// eslint-disable-next-line no-unused-vars
+import { ITimelineItem, TimelineItemAction, TimelineItemFirestoreConverter } from 'src/models/requests/timeline';
 
-import { IgetTimeline } from './types';
+const getRequest = async (requestRef: RequestRefType): Promise<IRequest> => {
+  console.log('abcd');
+  let requestData;
+    await firestore
+      .collection('requests')
```

Comment:
The above two lines and the below line need to be replaced with firestore.doc(requestRef.path) I believe.